Zend/zend_language_scanner.l

```diff
@@ -1545,12 +1624,37 @@ NEWLINE ("\r"|"\n"|"\r\n")
 	return T_SR;
 }
 
+<ST_IN_SCRIPTING>"," {
+	if((in_class && in_const_list)) {
+		yy_push_state(ST_LOOKING_FOR_SR_NAME);
+	}
+	return ',';
+}
```

Comment:
What about `const FOO = [1, isset(BAR[0])];` will the `isset` be lexed as T_ISSET or as T_STRING? It should be a T_ISSET. We currently don't support any language constructors in constants I think, but that's just because they don't happen to be implemented, we may want to add them in the future.